### ISSUE TITLE
Command handler writes the command inside the files

### ISSUE THREAD

**Issue by Cyb3r-Monk** [cmu-sei/GHOSTS#113]
I'm simulating file server browse with the below handler.

```JSON
    {
      "HandlerType": "Command",
      "Initial": "",
      "UtcTimeOn": "08:00:00",
      "UtcTimeOff": "17:30:00",
      "HandlerArgs": {},
      "Loop": true,
      "TimeLineEvents": [
        {
          "TrackableId": null,
          "Command": "explorer \\\\labsrv01\\Public\\",
          "CommandArgs": [
            "exit"
          ],
          "DelayAfter": 900000,
          "DelayBefore": 0
        }
      ]
    }
```
When it gets run for the first time, it opens the folder. Then the next time it runs, it pastes the command randomly in a file. The example below is Powerpoint, but I also saw it pasting the command into email. 
<img width="730" alt="image" src="https://user-images.githubusercontent.com/26627224/185070553-af915d57-7d2b-42d6-b9e3-997dd5a33962.png">

Here is an Outlok email example:
<img width="437" alt="image" src="https://user-images.githubusercontent.com/26627224/185099767-ee95d21d-4921-470f-a0d2-a5f4aec0cc0f.png">

It seems like this happens when there is an open windows at the time Command handler runs. It pastes the command to the active window instead of opening cmd.exe and pasting it.

**Comment by sei-dupdyke** [comment]
The command writer uses focus to manipulate the cmd window, and it is trying to write when the focus has switched to another window. I'll look into fixing this today.

**Comment by sei-dupdyke** [comment]
I changed this to use standard in and out instead of a keystrokes library that was typing without having full control over which window was in focus. This new method seems to work far better. Released in 6.1.9.

**Comment by Cyb3r-Monk** [comment]
Thanks a lot! Does this apply to running PowerShell commands as well?

**Comment by sei-dupdyke** [comment]
Yea, should be no change in the timeline files too.

**Comment by Cyb3r-Monk** [comment]
After the first execution of the command, I see the below error. Then it stops execution the same commands(loop is enabled)
```
2022/08/23 11:09:27.959|Ghosts.Client.Handlers.Cmd.Ex|Command line: explorer \\labsrv01\Public\ with delay after of 900000
2022/08/23 11:09:32.973|Ghosts.Client.Handlers.Cmd.Ex|Command line: powershell -file C:\Users\suser\Desktop\test.ps1 with delay after of 30000
2022/08/23 11:09:35.992|Ghosts.Client.Handlers.Cmd..ctor|System.ObjectDisposedException: Cannot write to a closed TextWriter.
```

**Comment by sei-dupdyke** [comment]
This has been fixed in 6.1.10.
